### DISCUSSION TITLE
Can the coding agent write code, save to local environment?

### DISCUSSION THREAD

**Discussion by chiranjeeviga** [VRSEN/agency-swarm#75] [Q&A]
I created an agency with coding/developer capabilities. I asked it to write code for a web portal to upload files. Agency CEO finally confirms that the task is successful and the web portal development is complete, but I don't see any files nor a localhost URL to test. 

Can the agents via agency be able to create/save code files? like html, css and js locally to my machine?

**Comment by chiranjeeviga** [comment]
Already resolved in issues tab.

**Comment by SwapnaliDhotre12** [comment]
@chiranjeeviga can you please mention the issue number # 
Thank you!
